src/components/Tooltip/TooltipConstants.js

```diff
@@ -5,7 +5,8 @@ export const TOOLTIP_THEMES = {
   Share: "share",
   Private: "private",
   Surface: "surface",
-  Primary: "primary"
+  Primary: "primary",
+  White: "white"
```

Comment:
why the new name include the explicit color in its name? we always trying to choose names according to usage and not to explicit color.

Reply:
let's try to find a naming which indicate its usage purpose?